src/CallEntityWrapperInterface.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_content;
+
+use Drupal\Core\Datetime\DrupalDateTime;
+use Drupal\Component\Render\MarkupInterface;
+
+/**
+ * Interface for content entity wrappers.
+ */
+interface CallEntityWrapperInterface {
+
+  /**
+   * Call for proposals not available, used when opening date isn not defined.
```

Comment:
Update the comments on this properties, they refer to any call now.

Reply:
Also this particular comment has a typo ("isn")

Reply:
This is fixed.